tests/integration/test_tracing.py

```diff
@@ -0,0 +1,83 @@
+# Copyright 2025 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+"""Feature: Ingested traces are pushed to Tempo."""
```

Comment:
Is now a good time to add a TLS variation of this test?

Reply:
Done! Currently being skipped until Tempo pulls in the dependency, but I tried it with a locally-packed version and it works :)